benchmarks/test_benchmarks.py

```diff
@@ -0,0 +1,31 @@
+
```

Comment:
Should there be something in here to compile RustPython like `test_snippets.py` does?

Reply:
Probably, I don't know. Maybe also not. I think it is okay if it picks the latest release build.